test/verify/check-networkmanager-basic

```diff
@@ -42,6 +42,11 @@ class TestNetworkingBasic(NetworkCase):
         b.wait_in_text("#networking-graphs", "Transmitting")
         b.wait_in_text("#networking-graphs", "Receiving")
 
+        # screenshot assumes non-running firewalld and absent virbr0 (in particular, *3* interfaces)
```

Comment:
[example failure](https://logs.cockpit-project.org/logs/pull-16942-20220208-132850-836165a4-fedora-35/pixeldiff.html#TestNetworkingBasic-testBasic-network-main)

Reply:
This fix needs more work. [firewalld isn't a thing on coreos](https://logs.cockpit-project.org/logs/pull-16952-20220208-175753-3f7e58f7-fedora-coreos/log.html#283-2), [follow-up tests get confused](https://logs-https-frontdoor.apps.ocp.ci.centos.org/logs/pull-16952-20220208-175753-3f7e58f7-fedora-35/log.html), and it's [unstable](https://logs.cockpit-project.org/logs/pull-16952-20220208-175753-3f7e58f7-fedora-34/log.html#283). I'll take it out of this series.